bindings/wasm/src/account/update/create_method.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use js_sys::Promise;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::future_to_promise;
+
+use identity::account::MethodSecret;
+use identity::account::Update;
+use identity::account::UpdateError::MissingRequiredField;
+use identity::did::MethodScope;
+use identity::did::MethodType;
+
+use crate::account::method_secret::WasmMethodSecret;
+use crate::account::wasm_account::WasmAccount;
+use crate::did::WasmMethodScope;
+use crate::did::WasmMethodType;
+use crate::error::wasm_error;
+use crate::error::Result;
+use crate::error::WasmResult;
+
+#[wasm_bindgen(js_class = Account)]
+impl WasmAccount {
+  /// Adds a new verification method to the DID document.
+  #[wasm_bindgen(js_name = createMethod)]
+  pub fn create_method(&mut self, options: &CreateMethodOptions) -> Result<Promise> {
```

Comment:
Well, `any` can be `undefined`. But it's better if we can return undefined. I will create an issue later for this, since it occurs in other bindings too.